test/functional/test_site/expected/testCodeBlocks.html

```diff
@@ -32,6 +32,8 @@
         <pre><code class="hljs js"><span><span class="hljs-function"><span class="hljs-keyword">function</span> <span class="hljs-title">fourEmptyLinesBelow</span>(<span class="hljs-params"></span>) </span>{</span><span>&#x200B;</span><span>&#x200B;</span><span>&#x200B;</span><span>&#x200B;</span><span>} <span class="hljs-comment">// four empty lines above</span></span></code></pre>
         <p><strong>hljs span spanning multiple lines</strong></p>
         <pre><code class="hljs markdown"><span><span class="hljs-strong">*****</span></span><span>-----</span></code></pre>
+        <p><strong>start-from attr causes inline style to be set</strong></p>
+        <pre><code start-from="30" style="counter-set: line 29;" class="hljs markdown"><span><span class="hljs-strong">*****</span></span><span>-----</span></code></pre>
```

Comment:
We could also remove the `start-from` attribute from the tag

Reply:
Thanks! I'll remove the highlight lines attr also then